### DISCUSSION TITLE
Pregunta 2a sesión SITEMAP

### DISCUSSION THREAD

**Discussion by BrunoPCH** [Launch-X-Latam/MisionFrontEnd#71] [Q&A]
@romarpla Mission Commander ¿En el sitemap que nos mostraste tenía solo botones o también pantallas emergentes? ¿Qué componentes se ponen en el sitemap? Me gusto el formato que tenias, pero con las prisas no pude observar más. ¿Podrías compartir información de cómo generar un sitemap práctico y eficiente?

**Comment by JosafatJimenezB** [comment]
Puedes volver a ver la transmisión por si te perdiste algo que querías ver o puedes ver la grabación de la sesión

para ver las grabaciones consulta la discussion #22

**Reply by BrunoPCH** [reply]
Gracias! pero en la sesión solo abordó de forma muy rápida el sitemap y volví a ver la repetición (en esa parte)

Gracias por tu comentario

**Reply by JosafatJimenezB** [reply]
Si no lo encontraste puedes buscar mas sobre el tema para profundizar en lo que deseas
